app/components/discussion-thread-view/index.ts

```diff
@@ -0,0 +1,139 @@
+import { pick } from 'lodash';
+
+import template from './template.html!text';
```

Comment:
we lose HTML minification here but that's probably only a few bytes after server-side gzipping.